cranelift/filetests/filetests/runtests/atomic-rmw-subword-big.clif

```diff
@@ -18,12 +18,12 @@ block0(v0: i32, v1: i64, v2: i16):
     v6 = load.i32 big v3
     return v6
 }
-; run: %atomic_rmw_add_little_i16(0x12345678, 0, 0x1111) == 0x23455678
-; run: %atomic_rmw_add_little_i16(0x12345678, 0, 0xffff) == 0x12335678
-; run: %atomic_rmw_add_little_i16(0x12345678, 2, 0x1111) == 0x12346789
-; run: %atomic_rmw_add_little_i16(0x12345678, 2, 0xffff) == 0x12345677
+; run: %rmw_add_big_i16(0x12345678, 0, 0x1111) == 0x23455678
+; run: %rmw_add_big_i16(0x12345678, 0, 0xffff) == 0x12335678
+; run: %rmw_add_big_i16(0x12345678, 2, 0x1111) == 0x12346789
+; run: %rmw_add_big_i16(0x12345678, 2, 0xffff) == 0x12345677
```

Comment:
Yes, `test run` (used to) ignore the actual name of the function.

We have had similar issues when we started adding `test interpret` to the runtests, since it calls functions by their `ExternalName`, however the interpreter also isn't used in this file.